ios/src/TGLabelPickResult.h

```diff
@@ -0,0 +1,24 @@
+//
+//  TGLabelPickResult.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/02/16.
+//
+
+#import <Foundation/Foundation.h>
+#import "TGGeoPoint.h"
+
+typedef NS_ENUM(NSInteger, TGLabelType) {
+    TGLabelTypeIcon = 0,
+    TGLabelTypeText,
+};
+
+@interface TGLabelPickResult : NSObject
+
+- (id) initWithCoordinates:(TGGeoPoint)coordinates type:(TGLabelType)type properties:(NSDictionary *)properties;
+
+@property (readonly, nonatomic) TGGeoPoint coordinates;
```

Comment:
Nitpick: I like to be explicit with properties memory management. So this would read
`+@property (readonly, assign, nonatomic) TGGeoPoint coordinates;`

whereas the Dictionary below would read
`+@property (readonly, strong, nonatomic) NSDictionary* properties;`

This becomes important for NSStrings and a few other classes where the default, because it's an object, is `strong`, but the convention is to always declare `copy` for them to avoid weirdness with class clusters.